res/values/setup.xml

```diff
@@ -45,6 +45,11 @@
 
     <!-- Button -->
     <color name="button_text_color">#000000</color>
+
+    <!-- Multiselect backgrounds -->
+    <color name="action_mode_background">#757575</color>
```

Comment:
Not sure that are really needed in ```setup.xml```, ```color.xml``` might be good enough; @rperezb , what do you think?